BitwardenShared/Core/Platform/Services/Stores/AppSettingsStore.swift

```diff
@@ -906,6 +933,16 @@ extension DefaultAppSettingsStore: AppSettingsStore {
         fetch(for: .accountSetupVaultUnlock(userId: userId))
     }
 
+    func addUserAction(_ action: UserAction) {
+        var reviewPromptData = reviewPromptData ?? ReviewPromptData()
+        if let index = reviewPromptData.userActions.firstIndex(where: { $0.userAction == action }) {
+            reviewPromptData.userActions[index].count += 1
+        } else {
+            reviewPromptData.userActions.append(UserActionItem(userAction: action, count: 1))
+        }
+        self.reviewPromptData = reviewPromptData
```

Comment:
🎨 Should we move this logic to `ReviewPromptData` and just allow getting and setting the model here? That would keep the logic together and this focused on storing the data.